plugins/inputs/http_listener_v2/http_listener_v2_test.go

```diff
@@ -231,8 +234,10 @@ func TestWriteHTTP(t *testing.T) {
 	require.EqualValues(t, 204, resp.StatusCode)
 
 	acc.Wait(2)
-	hostTags := []string{"server02", "server03",
-		"server04", "server05", "server06"}
+	hostTags := []string{
+		"server02", "server03",
+		"server04", "server05", "server06",
+	}
```

Comment:
Could you please revert the change? Even though I agree it looks better with your change, let's try to minimize modified lines to get a more clear view of what _actually_ changed.

Reply:
Yeah will revert. Maybe you could consider introducing `gofumpt` as main linter? :)